src/layman/upgrade/upgrade_v1_12_test.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+
+from layman import app, settings
+from . import upgrade_v1_12
+from test import process_client
+from layman.common.prime_db_schema import util as db_util
+
+db_schema = settings.LAYMAN_PRIME_SCHEMA
+
+
+@pytest.mark.usefixtures('ensure_layman')
+def test_adjust_prime_db_schema_for_last_change_search():
+    workspace = 'test_adjust_prime_db_schema_for_last_change_search_workspace'
+    layer = 'test_adjust_prime_db_schema_for_last_change_search_layer'
+    map = 'test_adjust_prime_db_schema_for_last_change_search_map'
+
+    process_client.publish_workspace_layer(workspace, layer)
+    process_client.publish_workspace_map(workspace, map)
+    with app.app_context():
+        statement = f'ALTER TABLE {db_schema}.publications ALTER COLUMN updated_at DROP NOT NULL;'
+        db_util.run_statement(statement)
+        statement = f'update {db_schema}.publications set updated_at = null;'
+        db_util.run_statement(statement)
+
```

Comment:
- [ ] Before adjust_prime_db_schema_for_last_change_search(), assert that updated_at contain nulls.
- [ ] After adjust_prime_db_schema_for_last_change_search(), assert that both times are greater then time at line 16, and lower then time at line 24.
- [ ] After adjust_prime_db_schema_for_last_change_search(), assert that time of layer is lower then time of map.